code/datums/mood_events/generic_neutral_events.dm

```diff
@@ -0,0 +1,66 @@
+/datum/mood_event/burnt_thumb
+	description = "<span class='alloy'>I shouldn't play with lighters...</span>"
```

Comment:
There should be a specific css colour for these rather than using alloy, which is a clockcult colour and can be changed accordingly.